standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -2333,11 +2333,6 @@ private void create_table_core(final RawStore ms, final CreateTableRequest req)
       isReplicated = isDbReplicationTarget(db);
 
       firePreEvent(new PreCreateTableEvent(tbl, db, this));
-      // get_table checks whether database exists, it should be moved here
-      if (is_table_exists(ms, tbl.getCatName(), tbl.getDbName(), tbl.getTableName())) {
-        throw new AlreadyExistsException("Table " + getCatalogQualifiedTableName(tbl)
-            + " already exists");
-      }
```

Comment:
We need to check  the check of `is_table_exists ` of line 2275. Not just remove code here

Reply:
Double check is not necessary, transactions of different `pm`s do not affect each other, and the eventual consistency is guaranteed by the backend RDBMS.